addons/mail/static/tests/chat_window/chat_window_tests.js

```diff
@@ -386,6 +386,23 @@ QUnit.test(
     }
 );
 
+QUnit.test("Close active thread action in chatwindow on ESCAPE", async () => {
+    const pyEnv = await startServer();
+    pyEnv["discuss.channel"].create({
+        name: "General",
+        channel_member_ids: [
+            Command.create({ partner_id: pyEnv.currentPartnerId, is_minimized: true }),
+        ],
+    });
+    await start();
+    await click(".o-mail-ChatWindow-command", { text: "General" });
+    await click(".dropdown-item", { text: "Add Users" });
+    await contains(".o-discuss-ChannelInvitation");
+    triggerHotkey("Escape");
+    await contains(".o-discuss-ChannelInvitation", { count: 0 });
+    await contains(".o-mail-ChatWindow");
```

Comment:
Nitpicking: should assert `contains(".o-mail-ChatWindow")` after `start()`, just so that the `contains(".o-mail-ChatWindow")` at the very end can be understood as "it remains open"